src/app/p-common/p-common.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { PCommonRoutingModule } from './p-common-routing.module';
+import { MenubarComponent } from './menubar/menubar.component';
+
+@NgModule({
+  declarations: [ MenubarComponent ],
+  exports: [
+    MenubarComponent
+  ],
+  imports: [
+    CommonModule,
+    PCommonRoutingModule
+  ]
+})
+export class PCommonModule {
```

Comment:
a do czego to służy? Taki pojemnik na używane komponenty?

Reply:
Nom, żeby mój menubar można importować w innych modułach.

Reply:
Okej, to w takim razie routing module chyba nie będzie potrzebne